### ISSUE TITLE
Bad error message when unzip not installed

### ISSUE THREAD

**Issue by jonludlam** [ocaml/opam#4512]
When unzip is not installed the wrong error message is reported, e.g.:

```
$ opam upgrade
...
[ERROR] The sources of the following couldn't be obtained, aborting:
          - lwt.5.4.0
```

whereas `opam source` is more useful

```
$ opam source lwt.5.4.0
[lwt.5.4.0] found in cache
Fatal error:
"unzip": command not found.
```

```
# opam config report
# opam-version      2.0.7
# self-upgrade      no
# system            arch=x86_64 os=linux os-distribution=ubuntu os-version=20.10
# solver            builtin-mccs+glpk
# install-criteria  -removed,-count[version-lag,request],-count[version-lag,changed],-changed
# upgrade-criteria  -removed,-count[version-lag,solution],-new
# jobs              5
# repositories      1 (http) (default repo at e35eaaa7)
# pinned            1 (git)
# current-switch    4.11.1
```

**Comment by rjbou** [comment]
I guess that you are using opam 2.0.x. This issue was fixed in #4083. Here is the new output
```
opam install lwt --sw rm
The following actions will be performed:
  ∗ install lwt 5.4.0

<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>

#=== ERROR while fetching sources for lwt.5.4.0 ===============================#
OpamSolution.Fetch_fail("\"unzip\": command not found.")


<><> Error report <><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
┌─ The following actions failed
│ ⬇ fetch lwt 5.4.0
└─ 
╶─ No changes have been performed
```

At init, opam checks that unzip is installed in the system, it is part of the required tools, see `opam init --show-default-opamrc` `required-tools:` field.

**Comment by jonludlam** [comment]
Lovely thanks!
